### ISSUE TITLE
Write console command to re-calculate counter aggregate value

### ISSUE THREAD

**Issue by antonkomarev** [antonkomarev/github-profile-views-counter#71]
We need to create console command which will count lines in `storage/{username}-views` file and store them to `storage/{username}-views-count`.

Something like:
```sh
cat storage/{username}-views | wc -l | tr -d ' ' > storage/{username}-views-count
```

But we should iterate thru all the usernames in the storage directory. There are many counter files, so ls -la is hanging for a long time.

**Comment by antonkomarev** [comment]
It looks like this command can compute correct value:
```shell
awk 'END {print NR-1}' storage/{username}-views > storage/{username}-views-count
```

**Comment by Brikaa** [comment]
I believe `wc -l` does not count the new line at the end of the file
```bash
$ echo -e "hello\nthere" > test.txt && wc -l test.txt
2 test.txt
```

**Comment by antonkomarev** [comment]
Try

```bash
echo -e "hello\nthere\n" > test.txt && wc -l test.txt
```

**Comment by antonkomarev** [comment]
That's exactly how views are stored. We have a blank line on the end of the file.

**Comment by Brikaa** [comment]
Yes, but your example puts two blank lines at the end of the file.
![image](https://github.com/antonkomarev/github-profile-views-counter/assets/40431124/18a1bf05-a0b5-4b36-b267-796448525658)

**Comment by Brikaa** [comment]
Does the following script using [Perl's readdir()](https://perldoc.perl.org/functions/readdir) for listing the directory content hang? It shouldn't try to read the entire directory at once.
```perl
my $dir = "/app/storage";
opendir(my $dh, $dir) || die "Can't open $dir: $!";
while (my $f = readdir $dh) {
  print "$dir/$f\n";
}
closedir $dh;
```
If it doesn't we can build upon that.

If it hangs, we can consider fixing the user on each HTTP request. We can have a flag that determines if a certain username is fixed, and on each request we check if the flag exists. If it doesn't, we fix the user and create the flag.
